api-tests/api.test.js

```diff
@@ -117,10 +117,18 @@ const addr = `http://${config.server.httpAddress}`
 
 const youtubeOrTwitch = /^(youtube|twitch)$/
 
-test('the api is alive', () => {
+test('the api is listening', () => {
   return fetch(addr + '/')
 })
 
+test('the api is healthy', async () => {
+  const d = await fetch(addr + '/').then(d => d.json())
+  expect(d.data.ok).toBe(true)
+  expect(d.data.listen.http).toBe(config.server.httpAddress)
```

Comment:
Do we want to also check the https address here?

Reply:
Nope. It's not listening on https iirc.

Reply:
👍